backend/config/default.json

```diff
@@ -22,8 +22,12 @@
     "getokProd": {
       "endpoint": "https://api.nrs.gov.bc.ca/webade-api/v1"
     },
-    "mssc": {
-      "endpoint": "https://i1api.nrs.gov.bc.ca/cmsg-messaging-api/v1"
+    "ches": {
+      "tokenEndpoint": "redacted",
+      "emailEndpoint": "https://ches-master-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1/email",
+      "healthEndpoint": "https://ches-master-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1/health",
```

Comment:
There's only 2 endpoints needed for CHES, so would be more keys to have a base path + route with not much benefit (would have to change 1 thing instead of just 2). So probably not at this point. If more were used would nest it.